src/operations.pxi

```diff
@@ -312,8 +312,8 @@ class Operations(object):
         *fh* will by an integer filehandle returned by a prior `open` or
         `create` call.
 
-        This method must returns the number of bytes written. However, uuless
-        the file system has been mounted with the ``direct_io`` option, the file
```

Comment:
this is actually correct

Reply:
first, i can not understand the word "uuless" :(

second, if "uuless" == "unless" , then :
"unless bob has been outdoor **with** a umbrella, bob will wet"
or
"unless bob has been outdoor **without** a umbrella, bob will wet" ?
mybe i do not understand the unless clause (or syntax in english) :(